Yoast/YoastSEO.js
# Add links to SEO assessments

## Summary

This PR can be summarized in the following changelog entry:

* Adds links to relevant articles to all SEO assessments.

## Test instructions

This PR can be tested by following these steps:

Build a beta version using `trunk` for Yoast SEO and this branch for YoastSEO.js.

Trigger the following assessments, and check whether the feedback texts contain links to relevant articles. For each assessments, test all scenarios described and check the feedback text with the link.
* internalLinks
  * Don't add any links
  * Add a nofollowed internal link
  * Add a normal internal link.
  * Add a nofollowed and a normal internal link.
* introductionKeyword
  * Set a keyword and add a paragraph without the keyword.
  * Include the keyword in the paragraph.
* keyphraseLength
  * Don't set a keyword.
  * Set a keyword with > 10 words.
* keywordDensity
  * Add a keyword and a tet with more than 100 words.
  * Add the keyword to the text a number of times. Make sure to obtain the following densities:
    * Between 0 and 0.5
    * Between 0.5 and 2.5
    * Between 0.5 and 3.5
    * More than 3.5
* metaDescriptionKeyword
    * Add a keyword and don't include it in the meta description.
    * Add the keyword in the meta desription.
* metaDescriptionLength
    * Add a post without any metadescription.
    * Add a very short metadescription (under 120 characters).
    * Add more words to metadescription (between 120 and 156).
    * Add even more words to metadescription (above 156).
* outboundLinks
    * Add a post without any links to any external websites.
    * Refer from your post to an external website using a follow-link.
    * Change the follow-link into a no-follow-link.
    * Add another link (can be to the same external website) and make it follow.
* pageTitleWidth
    * Add no SEO title at all
    * Fill in a short SEO title (<400 pixels)
    * Add a few words to the SEO title so that the bullet turns green (between 400 and 600 pixels)
    * Add a few more words to the SEO title (>600 pixels) so that the bullet turns red
* subheadingsKeyword
    * Add a keyword. Add one or more subheadings and don't use the keyword in it.
    * Add the keyword in one or more of the subheadings.
* taxonomyTextLength
    * Make a new category page.
    * Add a very short text (less than 50 words). The assessment should return a red bullet.
    * Add a little more copy (between 50 and 100). The assessment should still return a red bullet.
    * Add a little more copy (between 100 and 125). The assessment should still return a red bullet.
    * Add a little more copy (125 and 150). The assessment should return an orange bullet.
    * Add a little more copy (more than 150 words). The assessment should return a green bullet.
* textCompetingLinks
    * Add a post and fill in a certain keyword.
    * Add a link with the keyword as the link text (the URL doesn't matter) The assessment should return a bad score.
* textImages
    * Add a text without images. The assessment should return a red bullet.
    * Add an image without alt tags. The assessment should return an orange bullet.
    * Add an alt tag. The assessment should return an orange bullet.
    * Add a keyword (in the keyword box). The assessment should return an orange bullet.
    * Add your keyword to the alt tag. The assessment should return a green bullet and "The images on this page contain alt attributes with the focus keyword."
* textLength
    * Add a very short text (5 words). The assessment should return a red bullet.
    * Add a little more copy (150 words). The assessment should still return a red bullet.
    * Add a little more copy (230 words). The assessment should still return a red bullet.
    * Add a little more copy (270 words). The assessment should return an orange bullet.
    *  Add a little more copy (310 words). The assessment should return a green bullet.
* titleKeyword
    * Add a keyword and a title that starts directly with that keyword - the assessment should return a green bullet.
    * Add a word before the keyword - the assessment should return an orange bullet.
    * Remove the keyword from the title - the assessment should return a red bullet.
* urlKeyword
    * Set a keyword and make sure it's not in the URL of the past.
    * Add the keyword to the URL.

Notes:
* No link has been added to the urlStopWords assessment, since that assessment will be removed soon.
* The following assessments already had links: keywordStopwords, urlStopWords, previouslyUsedKeywords.


Fixes #1493
